Svc/Framer/docs/sdd.md

```diff
@@ -199,7 +199,7 @@ The `eventLogger` instance sends event packets to the `framer` instance.
 **Topology 3: File packets:**
 
 <div>
-<img src="img/top/file.png" width=1000/>
+<img src="img/top/framer.png" width=1000/>
```

Comment:
See my comment above. If necessary we can call this one `framer-file` -- but again, why are we managing all file names in the entire F Prime repository in one global namespace? This is bad.